pkg/apis/core/v1/storagecluster.go

```diff
@@ -596,6 +596,7 @@ type ComponentImages struct {
 	Telemetry                  string `json:"telemetry,omitempty"`
 	MetricsCollector           string `json:"metricsCollector,omitempty"`
 	MetricsCollectorProxy      string `json:"metricsCollectorProxy,omitempty"`
+	PxRepo                     string `json:"pxrepo,omitempty"`
```

Comment:
Can we add this to the schema under `deploy/crds/` so it doesn't get pruned by k8s as unknown field?